[ovs offload] script name change ovs_offload_lnv.py to ovs_offload_lnw.py (#2)
ipdk-io/ipu-playbook: ovs_offload/README.md

# OVS Offload Scripts

## Introduction

- The ovs_offload_lnw.py script can be used on an host server connected with Intel® Infrastructure Processing Unit via PCIe.
- It creates the configuration to run the Linux Networking Recipe with OVS Offload on the ACC.
- It can be used to start infrap4d on the ACC and use the script generated p4rt-ctl rules to configure ACC Port representors for the Host IDPF interfaces and IPU Physical Ports.
- It can setup OVS bridges on the ACC using the port representors and configure VMs on the IDPF interfaces on the Host.

## Test Environment Setup

> - Before running script check if ssh as a root user is enabled and root user password is setup for ssh.

Author: Addressed this in the latest commit .

## Test Script:

### ovs_offload_lnw.py : (P4:fxp-net_linux-networking.p4, IPU SDK Release >= 1.7.0)

> 1. This is a python script : **ovs_offload/ovs_offload_lnw.py** that can be used with **P4: fxp-net_linux-networking.p4** for release 1.7.0

Reviewer: ```suggestion
1. This is a python script : **ovs_offload/ovs_offload_lnw.py** that can be used with **P4: fxp-net_linux-networking.p4** for release 1.7.0 and later
```

### 3. Setup p4rt-ctl pipe and add the ACC PR rules

- ACC Terminal 2 : Setup p4rt-ctl pipeline and setup the runtime rules.

```bash
[root@ipu-acc ovs_offload_lnw_scripts]# pwd
/opt/p4/p4sde/p4_test/ovs_offload_lnw_scripts

[root@ipu-acc ovs_offload_lnw_scripts]# ./3_acc_p4rt.sh
```

- ACC Terminal 2 : Dump the p4rt-ctl runtime rules.

```bash
[root@ipu-acc ovs_offload_lnw_scripts]# pwd
/opt/p4/p4sde/p4_test/ovs_offload_lnw_scripts

[root@ipu-acc ovs_offload_lnw_scripts]# ./4_acc_p4rt_dump.sh
```

> ### 4. Setup ACC environment for OVS

Reviewer 2: ```suggestion
### 4. Set up ACC environment for OVS
```

> - ACC Terminal 2 : Setup the OVS Environment.

Reviewer 2: ```suggestion
- ACC Terminal 2 : Set up the OVS Environment.
```

```bash
[root@ipu-acc ovs_offload_lnw_scripts]# pwd
/opt/p4/p4sde/p4_test/ovs_offload_lnw_scripts

[root@ipu-acc ovs_offload_lnw_scripts]# ./5_acc_setup_ovs.sh
```

> ### 5. Setup OVS Bridge Configuration

Reviewer 2: ```suggestion
### 5. Set up OVS Bridge Configuration
```

> - ACC Terminal 2 : Setup the OVS Bridge Config

Reviewer 2: ```suggestion
- ACC Terminal 2 : Set up the OVS Bridge Config
```

```bash
[root@ipu-acc ovs_offload_lnw_scripts]# pwd
/opt/p4/p4sde/p4_test/ovs_offload_lnw_scripts

[root@ipu-acc ovs_offload_lnw_scripts]# ./6_acc_ovs_bridge.sh
```

> ### 6. Setup VMs on the IPU Host and configure link partner interface

Reviewer 2: ```suggestion
### 6. Set up VMs on the IPU Host and configure link partner interface
```

> ### 7. Optional: Setup OVS Bridge for VXLAN

Reviewer 2: ```suggestion
### 7. Optional: Set up OVS Bridge for VXLAN
```

```

> ### OVS Offload VXLAN on 2 IPU Peers Connected back to back

Reviewer 2: ```suggestion
### OVS Offload VXLAN on Two IPU Peers Connected back-to-back
```
